BUILD.md

```diff
@@ -0,0 +1,39 @@
+Environment
+This documentation is intended for Ubuntu 20.4
```

Comment:
change `Ubuntu 20.4` to `Ubuntu 20.04` since this is no ubuntu Ubuntu 20.4 image